src/bot/events/intro-music.ts

```diff
@@ -67,18 +65,31 @@ export class IntroMusic {
       const now = new Date();
       if (
         Math.abs(
-          differenceInSeconds(now, virgin.intro_last_played ?? new Date(0)),
+          differenceInMilliseconds(
+            now,
+            virgin.intro_last_played ?? new Date(0),
+          ),
         ) >=
         (virgin.intro_song?.computed_timeout_ms ??
           configuration.audio.default_intro.timeout_ms)
       ) {
+        // channel only has 1 member
+        if ((new_state.channel?.members.size ?? 0) >= 2) {
```

Comment:
The bot joins after this line so shouldnt this be '>=1' ? or am i misunderstanding here?

Reply:
We're checking to see if we should count this play against the user's timeout, so we want to see if the user was in there on their own or not.

Reply:
Yes but why 2 and not 1? If they are in there by themselves should the members size be 1?

Reply:
We only want to set their `intro_last_played` if anyone else heard it, as in, there are 2 or more people in the chat.

Reply:
ahhh my bad im dumb i misunderstood.